### ISSUE TITLE
toSVG() - add option to exclude backround image

### ISSUE THREAD

**Issue by dmytropopov** [kangax/fabric.js#2394]
`.toSVG()` function always include `<image>` element with background image. For someone, it can be a useful feature, but in my scenario it is an unwanted behavior. Also, that `<image>` element is not passed to reviver function so that it is possible to strip it only after `toSVG()` call. Of course, it takes only 1 extra line of code for workaround, but it would be nice to have such an option.

**Comment by asturur** [comment]
Using reviver was very easy. Adding an option  to esclude the background from export was too much. There is also #1186 that asks something similar, we can do one for both.
